code/datums/keybinding/_keybindings.dm

```diff
@@ -9,13 +9,16 @@
 	var/keybind_signal
 
 /datum/keybinding/New()
+	if(!keybind_signal)
+		CRASH("Keybind [src] called unredefined down() without a keybind_signal.")
```

Comment:
Can you add a unit test that initializes all keybindings so we can preemptively detect if this will be triggered please